doc/source/tutorial/stats.rst

```diff
@@ -152,10 +152,8 @@ argument:
     >>> norm.rvs(size=3)
     array([-0.35687759,  1.34347647, -0.11710531])   # random
 
-.. TODO: Replace numpy.random HTML with `numpy.random` once NumPy updates doc
-
 Note that drawing random numbers relies on generators from
-`numpy.random <https://www.numpy.org/devdocs/reference/routines.random.html>`__
```

Comment:
I didn't verify this works but it's used below so I assume I'm safe.

Reply:
Yep that worked.